packages/orbit-components/src/SkipLink/SkipLink.stories.tsx

```diff
@@ -37,231 +38,257 @@ const Content = (
-                  <FlightDirect size="small" />
-                  <Text weight="bold">Prague PRG</Text>
-                </Stack>
-              </Stack>
+              </Tooltip>
+              <Card>
+                <CardSection
+                  expandable
+                  actions={
+                    <Button type="secondary" size="small">
+                      Edit
+                    </Button>
+                  }
+                >
+                  <Stack direction="row" align="center" justify="between" spacing="200">
+                    <CarrierLogo carriers={[{ code: "FR", name: "Ryanair" }]} size="large" />
+                    <Stack spacing="100">
+                      <Text size="small" type="secondary">
+                        Sat, Mar 31 Trip length: 1h55m
+                      </Text>
+                      <Stack direction="row" spacing="100" align="center">
+                        <Text weight="bold">London LHR</Text>
+                        <FlightDirect size="small" />
+                        <Text weight="bold">Prague PRG</Text>
+                      </Stack>
+                    </Stack>
+                  </Stack>
+                </CardSection>
+                <CardSection
+                  expandable
+                  actions={
+                    <Button type="secondary" size="small">
+                      Edit
+                    </Button>
+                  }
+                >
+                  <Stack direction="row" align="center" justify="between" spacing="200">
+                    <CarrierLogo carriers={[{ code: "FR", name: "Ryanair" }]} size="large" />
+                    <Stack spacing="100">
+                      <Text size="small" type="secondary">
+                        Sat, Mar 31 Trip length: 1h55m
+                      </Text>
+                      <Stack direction="row" spacing="100" align="center">
+                        <Text weight="bold">London LHR</Text>
+                        <FlightDirect size="small" />
+                        <Text weight="bold">Prague PRG</Text>
+                      </Stack>
+                    </Stack>
+                  </Stack>
+                </CardSection>
+                <CardSection
+                  expandable
+                  actions={
+                    <Button type="secondary" size="small">
+                      Edit
+                    </Button>
+                  }
+                >
+                  <Stack direction="row" align="center" justify="between" spacing="200">
+                    <CarrierLogo carriers={[{ code: "FR", name: "Ryanair" }]} size="large" />
+                    <Stack spacing="100">
+                      <Text size="small" type="secondary">
+                        Sat, Mar 31 Trip length: 1h55m
+                      </Text>
+                      <Stack direction="row" spacing="100" align="center">
+                        <Text weight="bold">London LHR</Text>
```

Comment:
Where is this whole block of code in modal? :D
It's not visible there at all (maybe there's something I didn't catch, but I am confused :D :D)

<img width="613" alt="image" src="https://github.com/user-attachments/assets/3f63ea59-ec4c-4b1b-afb8-444d4559801b">

Reply:
Fixed ✅